### DISCUSSION TITLE
install/faq/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#5747] [Announcements]
# install/faq/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/install/faq/

**Comment by wangyao980112** [comment]
在线安装拉取镜像会报请求过多，导致pod起不来

**Comment by ihuibin** [comment]
卸载 kind 不能删除 /tmp  目录 是删除 /tmp 目录下的文件
